tests-integration/tests/http.rs

```diff
@@ -336,6 +354,21 @@ pub async fn test_prom_http_api(store_type: StorageType) {
         .await;
     assert_eq!(res.status(), StatusCode::OK);
 
+    // series
+    let res = client
+        .get("/api/v1/series?match[]=demo&start=0&end=0")
+        .send()
+        .await;
+    assert_eq!(res.status(), StatusCode::OK);
+    assert_eq!(res.text().await, "{\"status\":\"success\",\"data\":[{\"ts\":\"1970-01-01 09:00:00+0000\",\"cpu\":\"1.1\",\"host\":\"host1\",\"memory\":\"2.0\"}]}");
```

Comment:
I think we can just confirm the response text contains `memory` etc. The column's order in text may not be consistent between tests.